src/signals/incident-management/containers/Dashboard/components/BarChart/BarChart.tsx

```diff
@@ -2,22 +2,21 @@
 // Copyright (C) 2023 Gemeente Amsterdam
 import vegaEmbed from 'vega-embed'
 
-import { StyledBarChart, Wrapper } from './styled'
+import { BarChartWrapper } from './styled'
 import { vegaConfigBarChart } from './vega-config-bar-chart'
 import { ModuleTitle } from '../ModuleTitle'
 
 export const BarChart = () => {
   // needs to be function later, when endpoint is available
   const nrOfIncidents = 422
   vegaEmbed('#bar-chart', vegaConfigBarChart, { actions: false })
-  //there should be two spaces between title and amount
   return (
-    <Wrapper>
+    <BarChartWrapper>
```

Comment:
I like it better when I can easily see which wrapper belongs to which component. If that's only clear because of the directory name, then I first have to look that up.

Reply:
Changed it.